public/js/user/showListProducts.js

```diff
@@ -50,7 +46,35 @@ function appendHtml(response) {
                   </div>\
                 </div>\
               </div>\
-            </li>'   
-    );
-  });
+            </li>';
+          });
+  return $('#products').html(html);
 }
+$(document).ready(function () {
+  $.get(url, function(response) {
+    appendHtml(response);
+  });
+  $(".filter-price").on("click", function () {
+    from = $('#from').val();  
+    to = $('#to').val();
+    if(url.indexOf('?') > 0) {
+      url += '&min_price='+ from + '&max_price='+ to;
+    } else {
+      url += '?min_price='+ from + '&max_price='+ to;
```

Comment:
Đoạn này `min_price='+ from + '&max_price='+ to` lặp code. Cho ra ngoài `if`